src/renderer/components/reader/ReaderOptions.tsx

```diff
@@ -139,7 +139,7 @@ export class ReaderOptions extends React.Component<Props> {
                     <span className={styles.slider_marker} >a</span>
                     <input type="range"
                         onChange={(e) => this.props.handleIndexChange(e, "fontSize")}
-                        id="taille_texte"
+                        id="text_length"
```

Comment:
The point of issue https://github.com/readium/readium-desktop/issues/218 was that the ID is duplicated in the markup.

Reply:
New issue, to be addressed outside of this PR: https://github.com/readium/readium-desktop/issues/398